src/test/java/hw2/Ex2/TestEx2.java

```diff
@@ -0,0 +1,202 @@
+package hw2.Ex2;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.Test;
+import org.testng.asserts.SoftAssert;
+
+import java.util.List;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public class TestEx2 {
+    @Test
+    public void ex2() {
+        WebDriverManager.chromedriver().setup();
+        WebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
+
+        driver.get("https://epam.github.io/JDI/index.html");
+        SoftAssert softAssert = new SoftAssert();
+
+        // Assert Browser title
+        assertEquals(driver.getTitle().trim(), "Home Page");
+        // -----------------------------------------------------------
+
+        // Assert User name expected "PITER CHAILOVSKII
+        driver.findElement(By.id("user-icon")).click();
+        driver.findElement(By.cssSelector("#name")).sendKeys("epam");
+        driver.findElement(By.xpath("//input[@id='password']")).sendKeys("1234");
+        driver.findElement(By.id("login-button")).click();
+        WebElement loginedUserName = driver.findElement(By.id("user-name"));
+        assertEquals(loginedUserName.getText().trim(), "PITER CHAILOVSKII");
+        // -----------------------------------------------------------
+
+
+        // Assert "Service" subcategory in the header
+        WebElement serviceFromHeader = driver.findElement(By.className("dropdown-toggle"));
+        serviceFromHeader.click();
+        softAssert.assertTrue(serviceFromHeader.isDisplayed());
+        // ----------------------------------------------------------------
+
+
+        // Assert "Service" subcategory in the left section
+        WebElement serviceFromLeftSection = driver.findElement(By.className("menu-title"));
+        serviceFromLeftSection.click();
+        softAssert.assertTrue(serviceFromLeftSection.isDisplayed());
+        // -----------------------------------------------------------------
+
+
+        // Open Service - > Different Elements Page
+        driver.findElement(By.className("dropdown-toggle")).click();
+        List<WebElement> header = driver.findElements(By.tagName("li"));
+        for (int i = 0; i < header.size(); i++) {
+            if (header.get(i).getText().equals("DIFFERENT ELEMENTS")) {
+                header.get(i).click();
+                break;
+            }
+        }
+
+        // Assert 4 checkboxes at "Different Eelements" page
+        List<WebElement> checkboxes = driver.findElements(By.xpath("//label[@class = 'label-checkbox']"));
+        assertEquals(checkboxes.size(), 4);
+        for (int i = 0; i < checkboxes.size(); i++) {
+            assertTrue(checkboxes.get(i).isDisplayed());
+        }
```

Comment:
Why do you use here **Hard Assertions**. Please fix this issues in the current class